src/components/ScenarioParameters/components/ScenarioParametersInputs/components/TableDeleteRowsDialog/TableDeleteRowsDialog.js

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { DontAskAgainDialog } from '@cosmotech/ui';
+import PropTypes from 'prop-types';
+
+export const TableDeleteRowsDialog = (props) => {
+  const { t } = useTranslation();
+  const { open, onClose, onConfirm, selectedRowsCount } = props;
+  const labels = {
+    title: t('genericcomponent.table.deleteRowsDialog.title', 'Delete selected lines?'),
+    body: t('genericcomponent.table.deleteRowsDialog.body', { count: selectedRowsCount }),
+    cancel: t('genericcomponent.table.deleteRowsDialog.cancel', 'Cancel'),
+    confirm: t('genericcomponent.table.deleteRowsDialog.confirm', 'Delete'),
+    checkbox: t('genericcomponent.table.deleteRowsDialog.checkbox', "Don't show this message again"),
+  };
+  return (
+    <DontAskAgainDialog
+      id="delete-rows-dialog"
+      onClose={onClose}
+      onConfirm={onConfirm}
+      open={open}
+      labels={labels}
+      confirmButtonProps={{ color: 'error' }}
+    ></DontAskAgainDialog>
+  );
+};
+
+TableDeleteRowsDialog.propTypes = {
+  /*
+   * Boolean value, show the dialog when set on true
+   */
+  open: PropTypes.bool.isRequired,
+  /*
+   * Function execute when close the dialog
+   */
+  onClose: PropTypes.func.isRequired,
+  /*
+   * Function execute when press the confirm dialog button
+   */
+  onConfirm: PropTypes.func.isRequired,
+  /*
+   * Lenght of lines selected in the table
+   */
+  selectedRowsCount: PropTypes.number.isRequired,
+};
```

Comment:
Normally we don't need comments here since we are not generating docs for web app's components

Reply:
DONE